packages/cli/src/databases/repositories/workflow.repository.ts

```diff
@@ -669,4 +685,12 @@ export class WorkflowRepository extends Repository<WorkflowEntity> {
 			},
 		);
 	}
+
+	async moveToFolder(workflowIds: string[], toFolderId: string) {
+		await this.createQueryBuilder()
```

Comment:
Considering this a simple query, maybe better to do:

`await this.update({ parentFolder: toFolderId === PROJECT_ROOT ? null : { id: toFolderId } }, { id: In(workflowsIds)) })`